src/components/Reports/TeamReport/TeamReport.jsx

```diff
@@ -23,6 +28,15 @@ import './TeamReport.css';
 import { ReportPage } from '../sharedComponents/ReportPage';
 import UserLoginPrivileges from './components/UserLoginPrivileges';
 
+const persistConfig = {
+  key: 'root',
+  storage,
+  serialize: (outboundState) => compressToUTF16(JSON.stringify(outboundState)),
+  deserialize: (inboundState) => JSON.parse(decompressFromUTF16(inboundState))
```

Comment:
Your code looks fine however can you please try to move the parser in another function so that we can add try catch in that function because sometimes parse function gives error because mismatch strings just in case I am adding the reference here also
  deserialize: (inboundState) => parser(decompressFromUTF16(inboundState))
};

const parser =(val)=>{
  try {
    return JSON.parse(val);
  } catch (error) {
     console.log(error);
  }
}

thank you

Reply:
Hi @Ramaruva I have taken your advice to move the parser to another function to handle potential errors that might arise during the parsing process. Thanks for reviewing the codes.